zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/calls.kt

```diff
@@ -214,6 +214,7 @@ internal class RealCallSerializer(
       return object : SuspendingZiplineFunction<T>(
         name = functionName,
         argSerializers = listOf(),
+        resultSerializer = Int.serializer(), // Placeholder; we're only encoding failures.
```

Comment:
There's a `NothingSerializer` coming in the next version of kotlinx.coroutines.

Reply:
Oooh want.